.github/workflows/refreshcerts.yaml

```diff
@@ -2,74 +2,61 @@ name: Refreshcerts
+          sudo chmod -R 777 /gluster/@/api/keys
           echo "${{secrets.LETSENCRYPT_PRIVKEY}}" > /gluster/@/api/keys/letsencrypt.key
           echo "${{secrets.ZEROSSL_PRIVKEY}}" > /gluster/@/api/keys/zerossl.key
           echo "${{secrets.GOOGLE_PRIVKEY}}" > /gluster/@/api/keys/google.key
-          python3 ./tools/acme_certs.py "$FQDN"
-          if certinfo -expiry "$FQDN".cert.pem | grep -q CN="$FQDN"; then
-            echo "Letsencrypt certificate validated"
-          else
-            echo "Failing over to Google"
-            python3 ./tools/acme_certs.py -g "$FQDN"
-            if certinfo -expiry "$FQDN".cert.pem | grep -q CN="$FQDN"; then
-              echo "Google certificate validated"
-            else
-              echo "Failing over to ZeroSSL"
-              python3 ./tools/acme_certs.py -z "$FQDN"
-              if certinfo -expiry "$FQDN".cert.pem | grep -q CN="$FQDN"; then
-                echo "ZeroSSL certificate validated"
-              else
-                echo "Unable to get valid cert from any source"
-                exit 1
-              fi
-            fi
-          fi
-          set -e
-          cp "$FQDN".cert.pem ./tools/build_virtual_environment/ve_base/contents/atsign/root/certs/cert.pem
-          cp "$FQDN".key ./tools/build_virtual_environment/ve_base/contents/atsign/root/certs/privkey.pem
-          cp "$FQDN".fullchain.pem ./tools/build_virtual_environment/ve_base/contents/atsign/root/certs/fullchain.pem
-          cp ./tools/build_virtual_environment/ve_base/contents/atsign/root/certs/*.pem \
-            ./tools/build_virtual_environment/ve_base/contents/atsign/secondary/base/certs/
+      # Install Python Libraries
+      - name: Install Python Libraries
+        run: |-
+          python3 -m pip install requests dnspython
+      # Run Python ACME script
+      - name: Run ACME script
+        run: |-
+          set +e
+          chmod -R 777 secondaries-scripts
+          cd secondaries-scripts && ./create_cert_workflow.sh vip.ve.atsign.zone
+          cp cert.pem ../tools/build_virtual_environment/ve_base/contents/atsign/root/certs/cert.pem
+          cp privkey.pem ../tools/build_virtual_environment/ve_base/contents/atsign/root/certs/privkey.pem
+          cp fullchain.pem ../tools/build_virtual_environment/ve_base/contents/atsign/root/certs/fullchain.pem
+          cp ../tools/build_virtual_environment/ve_base/contents/atsign/root/certs/*.pem \
+            ../tools/build_virtual_environment/ve_base/contents/atsign/secondary/base/certs/
+          cd .. && rm -rf  vip.ve.atsign.zone* secondaries-scripts
           git config --global user.name 'Getcert Action'
           git config --global user.email '41898282+github-actions[bot]@users.noreply.github.com'
+          set -e
         env:
-          GITHUB_API_TOKEN: ${{ secrets.MY_GITHUB_TOKEN }}
           DO_KEY: ${{ secrets.DO_KEY }}
+          gChat_url: ${{ secrets.GOOGLE_CHAT_WEBHOOK }}
 
-      - name: store logs and responses
```

Comment:
Why has this step been removed?

Reply:
@cpswan - The reason to remove these steps is to sync all cert renewal workflow with the latest script(create_cert_workflow.sh) and we can see error logs during cert generation in the action console.